docs/asciidoc/front_matter.adoc

```diff
@@ -1,8 +1,17 @@
+////
+TODO:
+* do we want to add a self_url to the JSON response from DRS objects?  This would make it much, much easier to construct the access method rather than using the hostname-based URI which requires manipulation and parsing.
+* comments from David to note in the PR conversation
+  - removing provider_code
+	- mandating all DRS servers to register new prefixes
+	- saying 'drs.' should be in the prefixes
+////
```

Comment:
Please remove from the spec. (I'm not sure there are still relevant, but if so we should track them elsewhere.)

Reply:
Yep, done!   I captured the first todo (self URL reference) in a ticket.